packages/marko-web-gtm/browser/index.js

```diff
@@ -11,4 +11,47 @@ export default (Browser) => {
   Browser.register('GTMTrackBusEvent', GTMTrackBusEvent, {
     provide: { EventBus },
   });
+
+  // Send emitted IdentityX events to the datalayer
+  window.dataLayer = window.dataLayer || [];
+  [
+    // Views
+    'identity-x-authenticate-mounted',
+    'identity-x-comment-stream-mounted',
+    'identity-x-comment-post-mounted',
+    'identity-x-comment-create-mounted',
+    'identity-x-login-mounted',
+    'identity-x-logout-mounted',
+    'identity-x-profile-mounted',
+    // Actions/submissions
+    'identity-x-authenticated',
+    'identity-x-comment-post-submitted',
+    'identity-x-comment-report-submitted',
+    'identity-x-comment-stream-loaded',
+    'identity-x-comment-stream-loaded-more',
+    'identity-x-login-link-sent',
+    'identity-x-logout',
+    'identity-x-profile-updated',
+    // Errors
+    'identity-x-authenticate-errored',
+    'identity-x-comment-post-errored',
+    'identity-x-comment-report-errored',
+    'identity-x-comment-stream-errored',
+    'identity-x-login-errored',
+    'identity-x-logout-errored',
+    'identity-x-profile-errored',
+  ].forEach((event) => {
+    EventBus.$on(event, (args) => {
+      if (window.IdentityX) {
```

Comment:
Would be a little cleaner as `if (!window.IdentityX) return;` then the data layer push